File: ex11/ex11_01/ex11_01.py
import time

def main():
    count_down_time = int(input("Enter count down time in seconds: "))
    for i in range(count_down_time, 0, -1):
        print(f"Time remaining (s): {i}")
        time.sleep(1)
    print("Time's up!")

if __name__ == "__main__":
    main()

File: ex11/ex11_01/ex11_01.cpp
#include <iostream>
#include <unistd.h>

int main() {
    int count_down_time;
    std::cout << "Enter count down time in seconds: ";
    std::cin >> count_down_time;
    while (count_down_time > 0) {
        std::cout << "Time remaining (s): " << count_down_time << std::endl;
        sleep(1);
        count_down_time--;
    }   
    std::cout << "Time's up!\n";
    return 0;
}
